follow_test.go

```diff
@@ -1537,7 +1537,7 @@ func Test_postFollowing(t *testing.T) {
 					body := `{
 						"errors": [
 							{
-								"message":"Sorry, that page does not exist, code:34"
```

Comment:
This change was made because golangci-lint was crashing with goconst errors, but I don't understand the behavior and would like to remove it from the linter if possible.